src/Store/SessionStore.php

```diff
@@ -119,6 +121,10 @@ public function delete($key)
      */
     public function getSessionKeyName($key)
     {
-        return $this->session_base_name.'_'.$key;
+        $key_name = $key;
```

Comment:
@lbalmaceda - Just added this check. If an empty base name is passed in, skip the underscore being added. Edge case but this will at least look better if someone really doesn't want anything prefixed.